net/librsync/Portfile

```diff
@@ -22,34 +25,16 @@ long_description    librsync is a free software library that implements \
                     wire-compatible with rsync 2.x, and is not likely to be \
                     in the future.
 
-homepage            http://librsync.sourceforge.net/
-master_sites        sourceforge:project/librsync/librsync/${version}
+homepage            http://librsync.sourceforg.net/
 
-checksums           sha256  6633e4605662763a03bb6388529cbdfd3b11a9ec55b8845351c1bd9a92bc41d6 \
-                    rmd160  39ca29334d0efabc0ee9e4d44abbe73a7d2fe831
+checksums           sha256  ef8ce23df38d5076d25510baa2cabedffbe0af460d887d86c2413a1c2b0c676f \
+                    rmd160  cb2c372c3dc550e6b3d090f761cce8bda7db87a9 \
```

Comment:
The current checksum standard for MacPorts ports is `rmd160`, `sha256`, and `size`.  Please add the last one for this port.